python/pylibcudf/pylibcudf/tests/test_string_combine.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+import pyarrow as pa
+import pyarrow.compute as pc
+import pylibcudf as plc
+
+
+def test_concatenate():
+    arr = pa.array(["a", "b"])
+    sep = pa.scalar("")
+    plc_table = plc.interop.from_arrow(pa.table({"a": arr, "b": arr}))
+    plc_result = plc.strings.combine.concatenate(
+        plc_table,
+        plc.interop.from_arrow(sep),
+        plc.interop.from_arrow(pa.scalar("")),
+        plc.interop.from_arrow(pa.scalar("")),
```

Comment:
Can we test null handling? In particular not replacing nulls as well as replacing nulls.

Reply:
I think this is resolved.